### PR TITLE
added home screen

### PR DESCRIPTION
**Pull request by kv-yan**

### CODE REVIEW

File: app/src/main/java/com/simply/birthdayapp/commonpresentation/components/image/NetworkImage.kt [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        contentDescription = null,
+    )
+}
+
+
+@Preview(showSystemUi = true)
+@Composable
+fun PreviewNetworkImage() {
+    NetworkImage(url = "https://s3-alpha-sig.figma.com/img/54a6/abbe/eadb372e3e013ab667401f489eaf9aa1?Expires=1730678400&Key-Pair-Id=APKAQ4GOSFWCVNEHN3O4&Signature=aSJVU84B4XWFErfqgde1x5dZzWHgiI3yEXUXfUK27YGcocPOcmF115XEogJj7cWWD~Acxg3PZfrJ9xKiBw7cJPAI6PX95iJcLbEdTfwIr~TBPkHtjoiDNLHMvxEOp5Ons6MWAj-MmhptPe-4S04XlCDNhKMy4j-9eZh-O0QIYCB475sLdsZrVFyqN6k8hATvKT1~puoaQtHdJocWgMrU-ZbRArCtVBe64MJ2te6eZDEFlrI1adnyNGTUwwQlU-fl9FN42HakQVPvtinEk3mcJjKhkIjsPeKWMR-ADNdoTkWCzOIXh6w4oWAOG5ulSBOVS~UfaLubBUe~3LjQv~Q2pw__")
+}
```
Comment (4rzumanyan) [inline app/src/main/java/com/simply/birthdayapp/commonpresentation/components/image/NetworkImage.kt:47]: Does it work?

File: app/src/main/java/com/simply/birthdayapp/commonpresentation/theme/TextStyle.kt [hunk 2]
```diff
@@ -40,6 +39,12 @@ val ButtonTextStyle = TextStyle(
     lineHeight = 24.42.sp
 )
 
+val BirthdayItemTextStyle = TextStyle(
+    fontSize = 20.sp,
+    fontWeight = FontWeight.W700,
+    fontFamily = FontFamily(Font(R.font.karma_medium)),
+    lineHeight = 24.42.sp
```
Comment (4rzumanyan) [inline app/src/main/java/com/simply/birthdayapp/commonpresentation/theme/TextStyle.kt]: 24.42 ???

File: app/src/main/java/com/simply/birthdayapp/main/home/data/repository/BirthdayRepositoryImpl.kt [hunk 3]
```diff
@@ -0,0 +1,10 @@
+package com.simply.birthdayapp.main.home.data.repository
+
+import com.simply.birthdayapp.main.home.data.getMockBirthdays
+import com.simply.birthdayapp.main.home.domain.repository.BirthdayRepository
+import kotlinx.coroutines.flow.flowOf
+
+class BirthdayRepositoryImpl : BirthdayRepository {
+    // TODO: replace flow of to api call
+    override fun getBirthdays() = flowOf(getMockBirthdays())
+}
```
Comment (4rzumanyan) [inline app/src/main/java/com/simply/birthdayapp/main/home/data/repository/BirthdayRepositoryImpl.kt]: Write a normal implementation with api requests for later.

File: app/src/main/java/com/simply/birthdayapp/main/home/presentation/HomeViewModel.kt [hunk 4]
```diff
@@ -0,0 +1,11 @@
+package com.simply.birthdayapp.main.home.presentation
+
+import androidx.lifecycle.ViewModel
+import com.simply.birthdayapp.main.home.domain.usecase.GetBirthdayUseCase
+
+class HomeViewModel(
+    private val birthdayUseCase: GetBirthdayUseCase
+) : ViewModel() {
+    val birthdays = birthdayUseCase.invoke()
+
```
Comment (4rzumanyan) [inline app/src/main/java/com/simply/birthdayapp/main/home/presentation/HomeViewModel.kt]: Write a normal implementation for later.

File: app/src/main/java/com/simply/birthdayapp/commonpresentation/components/image/NetworkImage.kt (hunk 1 above)
Comment (kv-yan) [reply app/src/main/java/com/simply/birthdayapp/commonpresentation/components/image/NetworkImage.kt:47]: Of course

**Review by 4rzumanyan — DISMISSED** [review]

**Review by 4rzumanyan — APPROVED** [review]

**Pull request merged**